crates/subspace-networking/src/node_runner.rs

```diff
@@ -1302,7 +1302,7 @@ where
                 result_sender,
             } => {
                 if !self.swarm.behaviour().gossipsub.is_enabled() {
-                    panic!("Gossibsub protocol is disabled.");
+                    panic!("Gossipsub protocol is disabled.");
```

Comment:
This is the other place where we panic in async code. Is it a logic error, or should we replace it with returning an error to the (async) caller?

Reply:
This is a hard one. On one hand this is not a local decision that can be made in this context, on the other hand if we get to this point, it is a very fundamental application composition that I would say justified crashing the whole app since a lot of things will not work properly if this is actually the case.

Ultimately though we should probably simply remove gossipsub until we actually use it (it can be enabled or disabled during instantiation of the networking and it is currently disabled in 100% of cases and was for a very long time).